monkey/monkey_island/cc/main.py

```diff
@@ -35,26 +36,29 @@
 
 def main(should_setup_only=False, server_config_filename=DEFAULT_SERVER_CONFIG_PATH):
     logger.info("Starting bootloader server")
+
+    data_dir = env_singleton.env.get_config().data_dir_abs_path
```

Comment:
Wouldn't it be better to improve the readability of config param fetching (like create a method `env_singleton.get_data_dir_path()`) and use it everywhere else? Right now we get the `data_dir` from a singleton and pass it through multiple layers (main.py -> app.py -> local_run.py -> run_local_monkey(which should be a service)). If the singleton param list grows, will we be passing each of them as parameters? For example in `init_api_resources(api, data_dir)`, it's not really clear what data directory has to do with resources. And it actually has nothing to do with resources, because resources are responsible for request handling and data dir is used to launch monkey, which is a service.

Reply:
TL;DR I think we should do the same with local monkey run service as we do with PBA service.